gum/arch-arm64/gumarm64writer.c

```diff
@@ -982,3 +1044,94 @@ gum_arm64_writer_describe_reg (GumArm64Writer * self,
   }
   ri->index = ri->meta - GUM_MREG_R0;
 }
+
+void
+gum_arm64_writer_put_push_all_registers (GumArm64Writer * self)
```

Comment:
How about `{push,pop}_all_x_registers()`? So it doesn't sound like it's implying that it includes the Q registers also.